pkg/storage/types.go

```diff
@@ -57,3 +57,11 @@ type MetricsPoint struct {
 	// MemoryUsage is the working set size. Unit: bytes.
 	MemoryUsage resource.Quantity
 }
+
+func cpuUsageOverTime(last, prev MetricsPoint) resource.Quantity {
```

Comment:
How about moving this to an utils file? I don't feel like it belongs here, where all the types are defined.

Reply:
I specificly tried to avoid using "utils" as a antypattern to dump of all function that don't fit anywhere else. Decided on `types.go` based on rule that the closer connected function/structs are, closer they should be located in source tree. I don't expect anyone to use MetricsPoint without using cpuUsageOverTime and vice versa.

Reply:
SG, thank you for the clarification :+1: